app.js

```diff
@@ -4,17 +4,26 @@ const app = express();
 const {
   viewAllCategories,
   viewAllReviews,
+  viewReviewById,
 } = require("./controller/controller");
 
 app.get("/api/categories", viewAllCategories);
 app.get("/api/reviews", viewAllReviews);
+app.get("/api/reviews/:review_id", viewReviewById);
 
-app.use((err, request, response, next) => {
+app.use((err, req, res, next) => {
   console.log(err);
-  response.status(500).send({ msg: "Internal server error" });
+  res.status(500).send({ msg: "Internal server error" });
 });
-app.use((request, response, next) => {
-  response.status(404).send({ msg: "Route Does Not Exist" });
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+app.use((req, res, next) => {
+  res.status(404).send({ msg: "Route Does Not Exist" });
   next(err);
 });
 
```

Comment:
I think you could consider splitting off these into a different file to keep it all neat at this point. An error handling file is always a good way to organise it all off :)